lib/md001.mjs

```diff
@@ -11,7 +11,12 @@ export default {
   "tags": [ "headings" ],
   "parser": "micromark",
   "function": function MD001(params, onError) {
-    let prevLevel = Number.MAX_SAFE_INTEGER;
+    let prevLevel = frontMatterHasTitle(
```

Comment:
Nitpick, but could you please assign the result of frontMatterHasTitle to its own const boolean? Just to clean up the ternary so it can be single-line and more obvious.